README.md

```diff
@@ -46,6 +46,10 @@ Contributions are very welcome. Please see [CONTRIBUTING.md](https://github.com/
 - [Xfive](https://github.com/xfiveco/front-end-test-project) Front-end test project used in recruitment.
 - [Yodlr](https://github.com/yodlr/frontend-code-challenge) Code/design challenge for front end engineering applicants.
 
+## Java
+
+- [WorldTicket](https://bitbucket.org/worldticket/recruitment-tests/src/default/) Front-end test project used in recruitment.
```

Comment:
It looks good but could you update the description, please? Something like "Create a simple web application for currency conversion." would be better.

Reply:
I've just updated it :)